.github/workflows/test.yml

```
@@ -69,6 +69,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   build-website:
     name: Run D3 Build of website for github pages
+    if: github.ref == 'refs/heads/main' # only run on main
```

@AshleySetter, I'd recommend building the website on every branch/push, unassuming it doesn't take too long.

After all, it's good to know that the website can be built without errors before merging a PR (and being able to download the website when reviewing a PR).

---

Hmmm, GitHub Actions is free, so :shrug:

You could also do something like the following, so it will still run on a PR, but not on a push:

```suggestion
    # only run on push to main or on a PR
    if: github.ref == 'refs/heads/main' || github.event_name == "pull_request" 
```